### DISCUSSION TITLE
Aggrid styling

### DISCUSSION THREAD

**Discussion by Shahin-rmz** [zauberzeug/nicegui#552] [Q&A] [answered]
Hello, I just show a dataframe with a aggrid.
it is about 200 rows.
My question is, how can I set height to full screen or make it out of `iframe`
here is how my code look like.
```python
data_dict = {
    'columnDefs': [
        {'headerName': col, 'field': col} for col in df.columns
    ],
    'rowData': df.to_dict('records'),
    'rowSelection': 'multiple',
}


print(data_dict)

ui.aggrid(data_dict).classes('max-h-999')
```
and here is how my table looks like

![image](https://user-images.githubusercontent.com/32805117/226132634-413dd767-2f62-4436-a679-ccea7df56c51.png)

but I want it to be full screen.

**Comment by falkoschindler** [comment, marked as answer]
`max-h-999` is not a Tailwind class. That's why it is not working. See [here](https://tailwindcss.com/docs/height) for available classes.

Making an HTML fullscreen is actually not that trivial. But you can set the height to 100% of the viewport height. In order to incorporate the 16px margin, you can subtract 32px from that viewport height:

```py
ui.aggrid({}).style('height: calc(100vh - 32px)')
```

**Comment by Shahin-rmz** [comment]
Thanks.
Sorry, I was not very familiar with front end CSS and other things.
